cmd/ltctl/report.go

```diff
@@ -90,6 +90,11 @@ func RunGenerateReportCmdF(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return fmt.Errorf("could not parse output: %w", err)
 		}
+
+		if !output.HasMetrics() {
+			return fmt.Errorf("no active deployment found, use the `--prometheus-url` flag. If you have a local or manually deployed Prometheus server running")
```

Comment:
This is actually the same sentence! Sorry I wasn't clear before, but what I meant with the message is that if you have deployed Prometheus yourself (not with the tool), then you need to use the `--prometheus-url` flag. I think this is clear, but I'm open to other suggestions:
```suggestion
			return fmt.Errorf("no active deployment found, use the `--prometheus-url` flag if you have a local or manually deployed Prometheus server running")
```

Reply:
I should have removed the capital letter obviously while removing `\n`.
 In my opinion, this message is quite clear:
`no active deployment found, use the `--prometheus-url` flag if you have a local or manually deployed Prometheus server running`